### ISSUE TITLE
Deploying ordering org with BFT consensus by generating artifacts using CA - not working

### ISSUE THREAD

**Issue by cclchd-soundaryaayyappan** [hyperledger/fabric-samples#1285]
**Bug Description:**
Encountered an issue in `fabric-samples` version 3.0 where deployment of ordering org with BFT consensus is providing error at generating artifacts step using CA.

**Steps to Reproduce:**
Run generate_artifacts.sh script with arguments included like `./generate_artifacts -o BFT -ca`

**Expected Behavior:**
It should generate crypto-config and channel-artifacts required for the deployment.

**Actual Behavior:**
Resulted in below error
![Image](https://github.com/user-attachments/assets/99400035-2230-417e-a5ca-402c4b59bace)

**Environment:**
- fabric-samples version: 3.0
- Hyperledger Fabric version: 3.0
- OS: Ubuntu 22.04

**Comment by bestbeforetoday** [comment]
The README for the test-network does say:

> Note that currently this sample does not yet support the use of consensus type BFT and CA together

Therefore I don't think this behaviour is a bug. It certainly is a valid feature request though.

**Comment by cclchd-soundaryaayyappan** [comment]
@bestbeforetoday  Thanks for your response. Can you please share the link of the README where the mentioned statement is present?

**Comment by bestbeforetoday** [comment]
Certainly. It is in [test-network/README.md](https://github.com/hyperledger/fabric-samples/tree/main/test-network#readme).
